bundle/manifests/limitador-operator.clusterserviceversion.yaml

```diff
@@ -144,6 +144,7 @@ spec:
           template:
             metadata:
               labels:
+                app: limitador
```

Comment:
looks like the change did not make it to the bundle manifests. Run `make bundle`